README.md

```diff
@@ -1,29 +1,60 @@
 # diva-js-reference-3p-frontend
 
-This repository contains an example/reference frontend implementation using [diva-irma-js](https://github.com/Alliander/diva-irma-js)
-DIVA is an SDK to easily integrate [IRMA attributes](https://privacybydesign.foundation/irma-controleur/) into NodeJS based applications.
+This repository contains an example/reference frontend implementation to show a nice GUI for [diva-js-reference-3p-backend](https://github.com/Alliander/diva-js-reference-3p-backend)
+that uses the DIVA SDK [diva-irma-js](https://github.com/Alliander/diva-irma-js) to easily integrate [IRMA attributes](https://privacybydesign.foundation/irma-controleur/) into NodeJS based applications.
```

Comment:
No change in this PR, but maybe link to the English IRMA page? https://privacybydesign.foundation/irma-verifier/ 

Also maybe add a link to the website itself (without 'hiding' the URL) at the bottom of this paragraph?